lua/co-author/init.lua

```diff
@@ -22,9 +22,7 @@ co_author.list = function()
         vim.ui.select(items, { prompt = 'Select Co-Author' }, function(item, _)
             if item ~= nil then
                 local string = 'Co-authored-by: ' .. item
-                local cursor_position = vim.api.nvim_win_get_cursor(0)
-                local line = cursor_position[1] + 1
```

Comment:
the +1 here is what was causing the issue.

When debugging locally doing print(cursor_position_line, cursor_position_column)

it was printing always 2 0 wether it was lazygit or git commit UI so the issue was not getting the data but applying it when calling `vim.api.nvim_buf_set_lines(0, line, line, true, { string })`